packages/example/pages/api/auth/[...nextauth].ts

```diff
@@ -44,6 +44,7 @@ export function getAuthOptions(req: IncomingMessage): NextAuthOptions {
             id: siwe.address,
           };
         } catch (e) {
+          console.error('siwe authorization failed', e);
```

Comment:
Is this needed ?

Reply:
Noticed we weren't catching any logs in case the server-side verification failed, so I think this will be useful in the future; only in the example dapp.